### DISCUSSION TITLE
Provide files to the customers as service

### DISCUSSION THREAD

**Discussion by EternalTheGod** [Paymenter/Paymenter#353] [Ideas]
Could be useful to sell products like people do on builtbybit / sourcexchange / polymart, where the client will be able to download a file after they've purchased it.

You can either upload the file via paymenter itself, provide the path of the file inside of the machine ( /root/products/myExampleProduct.zip) or provide direct download link for the file, saving up space from the server by hosting the file elsewhere (mediafire etc.)

**Comment by finnjosten** [comment]
I myself don't like the software download thing but I do agree it would be nice after a purchase a file can be downloaded, maybe in comby with another product or just on its own
